callow.py

```diff
@@ -91,22 +92,22 @@ def crack(username, usersel, passsel, passlist, website):
         print(color.RED + '\n[!] '+color.WHITE + 'ChromeDriver binary not found')
         exit()
     browser.get(website) # Open target website
```

Comment:
Why not just wait 3 seconds after calling `get()`?

Reply:
I was thinking implicitly waiting 3 seconds may **slow the brute force.**
Whereas [explicitly waiting](https://www.geeksforgeeks.org/explicit-waits-in-selenium-python/) does not slow the brute force as much and counters many websites that have a redirect after finishing the login form and if incorrect redirect back to the login form..
In either case, redirecting or not, explicit waits do not have a dramatic impact on performance and simply does a check for the condition every 500 milliseconds.

Reply:
> Why not just wait 3 seconds after calling `get()`?

I was thinking implicitly waiting 3 seconds may slow the brute force.
Whereas [explicitly waiting](https://www.geeksforgeeks.org/explicit-waits-in-selenium-python/) does not slow the brute force as much and counters many websites that have a redirect after finishing the login form and if incorrect redirect back to the login form..
In either case, redirecting or not, explicit waits do not have a dramatic impact on performance and simply does a check for the condition every 500 milliseconds.